x/auctions/types/params.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import "fmt"
+
+var ParamsKey = []byte("Params")
+
+func NewParams(
+	auctionEnabled bool,
+) Params {
+	return Params{
+		EnableAuction: auctionEnabled,
+	}
+}
+
+func DefaultParams() Params {
+	return Params{
+		EnableAuction: true,
+	}
+}
+
+func validateBool(i interface{}) error {
+	_, ok := i.(bool)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+
+	return nil
+}
+
+func (p Params) Validate() error {
+	return validateBool(p.EnableAuction)
+}
```

Comment:
Why are these required? How can be different than a boolean?

Reply:
Not sure if it's possible to make a raw transaction to post something different than a `bool` but it's checked on all other modules.